src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -47,6 +47,23 @@ public AuthenticationTests()
             }
         }
 
+        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))] // Managed implementation connects successfully.
```

Comment:
You're right: connecting when there are no auth schemes supported doesn't sound like the right thing to do for the managed implementation.
I only duplicated the test above with all attributes. @stephentoub @Priya91 do you have more info on why we have that condition and not an ActiveIssue?